packages/qwik-city/src/runtime/src/types.ts

```diff
@@ -362,17 +362,27 @@ export type FailOfRest<REST extends readonly DataValidator[]> = REST extends rea
   : never;
 
 /** @public */
-export type ValidatorErrorKeyDotNotation<T, Prefix extends string = ''> = T extends object
-  ? {
-      [K in keyof T & string]: T[K] extends (infer U)[]
-        ? U extends object
-          ? `${Prefix}${K}[]` | `${Prefix}${K}[]${ValidatorErrorKeyDotNotation<U, '.'>}`
-          : `${Prefix}${K}[]`
-        : T[K] extends object
-          ? ValidatorErrorKeyDotNotation<T[K], `${Prefix}${K}.`>
-          : `${Prefix}${K}`;
-    }[keyof T & string]
-  : never;
+export type IsAny<T> = 0 extends T & 1 ? true : false;
```

Comment:
I think this is just a util type similar to the `Prettify` type above and should not be exported.